docs/data/horizon/README.mdx

```diff
@@ -5,7 +5,7 @@ sidebar_position: 10
 
 :::info
 
-Starting on August 1, 2024, SDF will be truncating historical data on SDF-owned Horizon to one year. This update allows us to optimize performance and ensure a streamlined experience for all users. We encourage you to explore third-party ecosystem providers of Horizon, which may provide a longer history retention window as well as other features.
+On August 1, 2024, the SDF truncated historical data on its Horizon instances to one year. This update allows us to optimize performance and ensure a streamlined experience for all users. We encourage you to explore third-party ecosystem providers of Horizon, which may provide a longer history retention window as well as other features.
```

Comment:
Only thing I might quibble with is the `the` you've added in `... 2024, the SDF ...`. It doesn't quite "fit" the way we've (generally) used `SDF` more like a proper noun (almost like a name lol).

I think we should remove that, but I'm **certainly** not planning to die on that hill hehe

Reply:
It's a material style choice imo. The way I thought about it was if I was vocally saying "_the_ Stellar Dev Foundation." Certainly it's been used [both](https://github.com/stellar/stellar-docs/blob/90e01c94a7ed6982bf1b5b680ac54c02a9f9cc74/docs/data/hubble/README.mdx?plain=1#L22) [ways](https://github.com/stellar/stellar-docs/blob/90e01c94a7ed6982bf1b5b680ac54c02a9f9cc74/openrpc/scripts/build.mjs#L149) throughout. 💬

For me, the larger question is how do we frame the Foundation as the single network participant that it is. Namely, while you, at the Foundation, may review this PR, the underlying network infrastructure always stays in the hands of actual validators. 🌐 As we all know, this leaves the SDF as a single network peer, which I think we ought focus on by continually [identifying](https://github.com/stellar/stellar-docs/issues/1041) its limited foundational role (e.g. via isolating _the_ introducing article).[^ind]

[^ind]: More to the point, this begins the necessary process of decentralizing not only the underlying `core` tech stack, but also the community documentation which is so key to permissionlessly educating the next billion about the power of Stellar. 🌌

Reply:
Does that make sense? Feel free to lmk your initial reaction and perhaps we could move overarching items into a discussion. 💭 Frankly, I think this should be included in the SDF's [404 Horizon response](https://horizon.stellar.org/transactions/90de1ff887cd708e1d30db0f17e9a0f6e646f86b5c04154d87957c20127080fc) `detail` rather than the definitive network docs.

Reply:
I think we're okay to leave the `the` in there. Let's merge it!